assets/js/phoenix_live_view.js

```diff
@@ -1273,7 +1273,7 @@ export class View {
       type: "form",
       event: phxEvent,
       value: serializeForm(inputEl.form, {_target: e.target.name}),
-      cid: this.targetComponentID(inputEl)
+      cid: this.targetComponentID(inputEl.form)
```

Comment:
Thanks for the PR!  Safe to say `MyForm` is the parent "Component" (and not just a plain LiveView) and `AutoCompleteLive` is a child "Component"?

Reply:
in my case `MyForm` was a component (not LiveView) but i assume the same behaviour would occur with LiveView `MyForm` as well

Reply:
unless the `AutocompleteLive` is a stateless component 🤔, in my case it was stateful

Reply:
We are working on this! We want to repurpose "phx-target" so it includes a CID. The idea is that all events go to the parent liveview unless they include phx-target=CID. The fix should be out shortly. Thank you!